batch7crud-roll7/src/main/java/com/lftechnology/batch7crud/service/StudentServices.java

```diff
@@ -12,8 +15,15 @@
 public class StudentServices {
   private StudentDataAccess studentDao = new StudentDataAccess();
 
-  public void addNew(Student student) throws DataException {
-    studentDao.addNew(student);
+  public void addNew(Student student) throws DataException, ValidatorException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: !unknownSymbol!, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)